linguatec_lexicon/management/commands/exportdata.py

```diff
@@ -0,0 +1,104 @@
+import json
+import pandas as pd
+
+from django.core.exceptions import ValidationError
+from django.core.management.base import BaseCommand, CommandError
+
+from linguatec_lexicon.models import (
+    Entry, Example, Lexicon, GramaticalCategory, VerbalConjugation, Word)
+
+import csv
+import argparse
+import os.path
+
+def get_src_language_from_lexicon_code(lex_code):
+    return lex_code[:2]
+
+def get_dst_language_from_lexicon_code(lex_code):
+    return lex_code[3:]
+
+
+
+class Command(BaseCommand):
+    def add_arguments(self, parser):
+        parser.add_argument(
+            'lexicon_code', type=str,
+            help="Select the lexicon code where data will be exported from",
+        )
+        parser.add_argument(
+            'output_file', type=str,
+            help='Name of file where data will be written to. (must be a csv file)'
+        )
+
+
+    def handle(self, *args, **options):
+        self.lexicon_code = options['lexicon_code']
+        self.output_file = options['output_file']
+        # check that a lexicon with that code exist
+        try:
+            src = get_src_language_from_lexicon_code(self.lexicon_code)
+            dst = get_dst_language_from_lexicon_code(self.lexicon_code)
+
+            self.lexicon = Lexicon.objects.get(src_language = src, dst_language = dst)
```

Comment:
```suggestion
            self.lexicon = Lexicon.objects.get(code=self.lexicon_code)
```

Reply:
About this I have tried and it is not possible because code is a property and can only be used to get it but no to do searchs of queryset, maybe should we change the property code in the model?